database/seeders/ProviderSeeder.php

```diff
@@ -54,6 +55,7 @@ public function run(): void
             ["name" => VeoDataImporter::getProviderName(), "color" => "#000000"],
             ["name" => WindDataImporter::getProviderName(), "color" => "#fffa00"],
             ["name" => WheeMoveDataImporter::getProviderName(), "color" => "#31682d"],
+            ["name" => HopDataImporter::getProviderName(), "color" => "#EA1821"],
```

Comment:
In this array all providers are in alphabetical order - why `Hop` is on the end of the array? Or maybe this order doesn't matter? Also all colors are lowercase, and this is uppercase

Reply:
Rearranged to alphabetical order